control_v2.proto

```diff
@@ -167,6 +167,8 @@ message StateAgentInfo {
   bool snapshot = 5;
   // Current running PID.
   int32 pid = 6;
+  // True when running as privileged.
+  bool privileged = 7;
```

Comment:
To keep nomenclature simple maybe we can flip this boolean (it would be coherent with the feature being introduced, the command line flags and the default value corresponds to the default mode of operation for agent. Something like
```suggestion
  // True when running unprivileged.
  bool unprivileged = 7;
```
The same would be for the agent_info down below privileged -> unprivileged
It doesn't change the behavior and using a privileged flag would work just as well, just trying to reduce cognitive load and possible confusion :sweat_smile:

Reply:
Thats true, I will switch the logic.

Reply:
I switched the logic in the latest commit.